packages/zoe/src/state.js

```diff
@@ -75,51 +76,60 @@ const makeInstallationTable = () => {
 
 // Instance Table
 // Columns: handle | installationHandle | publicAPI | terms | issuers
+// | issuerKeywordRecord | keywords
 const makeInstanceTable = () => {
   // TODO: make sure this validate function protects against malicious
   // misshapen objects rather than just a general check.
   const validateSomewhat = makeValidateProperties(
-    harden(['installationHandle', 'publicAPI', 'terms', 'issuers']),
+    harden([
+      'installationHandle',
+      'publicAPI',
+      'terms',
+      'issuers',
+      'issuerKeywordRecord',
+      'keywords',
```

Comment:
I was able to remove `issuers` successfully, but I think I like keeping keywords, since they would have to be regenerated quite often. What do you think?

Reply:
I'm fine with that. Thx for checking. Resolving.